packages/cozy-harvest-lib/src/components/cards/helpers.js

```diff
@@ -1,13 +1,30 @@
+import { formatLocallyDistanceToNow } from 'cozy-ui/transpiled/react/I18n/format'
+
+const DEFAULT_MINUTES = 300_000
+
+const getDifferenceInMinutes = date => {
+  return new Date(Date.now()).getTime() - new Date(date).getTime()
```

Comment:
Are we sure this is timezone safe?

Reply:
Avant tout, j'ai simplifié le calcul en
```js
return Date.now() - new Date(date).getTime()
``` 
Je ne pense pas qu'on aura des problèmes avec les fuseaux horaires, les 2 dates étant créées dans le même temps.
Mais ouvert à discussion :)

Reply:
`date` ne vient pas du job qui tourne sur le serveur?

Reply:
Vu en zoom, aucun problème 👍